sw/device/lib/testing/test_framework/ottf_isrs.c

```diff
@@ -10,6 +10,15 @@
 #include "sw/device/lib/runtime/ibex.h"
 #include "sw/device/lib/runtime/log.h"
 
+/**
+ * These weak symbols enable the OTTF ISRs to be used without the OTTF itself.
+ * This enables us to maintain one set of default ISRs for testing, while also
+ * enabling writing tests that do not make use of the OTTF. See the `crt_test`
+ * in `sw/device/tests/crt_test.c` for an example.
+ */
+OT_ATTR_WEAK uintptr_t pxCurrentTCB = (uintptr_t)NULL;
```

Comment:
Would it be possible to give these a pointer type rather than uintptr_t?

Reply:
Fixed.